### PR TITLE
Create irs-story issue template

### PR DESCRIPTION
**Pull request by jzbmw**
## Description

I want to add a issue template for an IRS - Story to manage the Project in Github Projects.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/irs-story [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+## Hints / Details
+<!--
+provide as much information regarding this request as possible
+-->
+- ...
+
+## Outcome / Acceptance Criteria
+<!--
+describe the Outcome the result of the story
+-->
```
Comment (mkanal) [inline .github/ISSUE_TEMPLATE/irs-story:21]: One idea is to define the "##Out of scope:" section explicitly - as if we do not have a structure for this everyone places and manages the out of scope individually. This is hard to cover.

File: .github/ISSUE_TEMPLATE/irs-story [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+name: Propose a Story
```
[the remark this answers is not in the document]
Comment (jzbmw) [reply .github/ISSUE_TEMPLATE/irs-story:2]: This was copied from the Features template. I think the Github standard for that is name https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository

File: .github/ISSUE_TEMPLATE/irs-story [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+<!--
+provide as much information regarding this request as possible
+-->
+- ...
+
+## Outcome / Acceptance Criteria
+<!--
+describe the Outcome the result of the story
+-->
+- ...
```
[the remark this answers is not in the document]
Comment (jzbmw) [reply .github/ISSUE_TEMPLATE/irs-story:22]: Dependencies will be part of hints. 
Within Tractus-X we can only reference to TX topics. No extra Dependencies Header will be initialized

File: .github/ISSUE_TEMPLATE/irs-story [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+name: Propose a Story
+about: propose a new story, new task
+---
```
[the remark this answers is not in the document]
Comment (jzbmw) [reply .github/ISSUE_TEMPLATE/irs-story:4]: Only labels is a valid type of meta information.

labels: Bug, Needs Triage

File: .github/ISSUE_TEMPLATE/irs-story [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+**I want** ... ,
+**so that** ... .
+
+## Hints / Details
+<!--
+provide as much information regarding this request as possible
+-->
+- ...
+
+## Outcome / Acceptance Criteria
```
[the remark this answers is not in the document]
Comment (jzbmw) [reply .github/ISSUE_TEMPLATE/irs-story:18]: For me it is the same. An Outcome of a story should relate exactly to a acceptance criteria. Let's discuss the differences on that. 
That is the Reason for it to be under the same header. 
I want this to be as lean as possible

[the remark this answers is not in the document]
Comment (jzbmw) [reply .github/ISSUE_TEMPLATE/irs-story:19]: added separate headers

**Review by mkanal — APPROVED** [review]

**Pull request merged**